tests/components/composer.test.ts

```diff
@@ -1206,6 +1207,26 @@ describe("composer", () => {
     expect(fixture.querySelectorAll(".o-grid div.o-composer")).toHaveLength(0);
   });
 
+  test("The composer should be closed before selecting headers", async () => {
+    composerEl = await typeInComposerGrid("=");
+    composerEl.dispatchEvent(new MouseEvent("click"));
+    expect(fixture.querySelectorAll(".o-grid-composer .o-composer")).toHaveLength(1);
+    await selectColumnByClicking(model, "C");
+    await nextTick();
```

Comment:
is the nextTick relevant here? I don't really see what it could change

Reply:
Deleting this line will make the test fail. I've added `await nextTick()` in `selectColumnByClicking` helper and this line is deleted.